src/crate/client/sqlalchemy/tests/array_test.py

```diff
@@ -39,7 +39,7 @@
 class SqlAlchemyArrayTypeTest(TestCase):
 
     def setUp(self):
-        self.engine = sa.create_engine('crate://')
+        self.engine = sa.create_engine('crate://cratedb.example.org/?cratedb-version=4.3.2')
```

Comment:
why is forcing the version here needed?

Reply:
I needed it to satisfy some test cases which validate behavior between CrateDB < 1.0.1 vs. CrateDB >= 1.0.1.
@mfussenegger had the same question, see https://github.com/crate/crate-python/pull/391#discussion_r879126382. In order to improve the situation _how_ this is implemented, I've added d91d088c.